### PR TITLE
Feature/authentication

### PR DESCRIPTION
**Pull request by MavOmidK**
Added logout functionality.

The angularfire createuser doesn't allow anything more than email + password

### CODE REVIEW

**Review by LeontiBrechko — APPROVED** [review]
Looks good! Thanks

File: src/app/app.component.ts [hunk 1]
```diff
@@ -1,11 +1,11 @@
-import { Component, ViewChild } from '@angular/core';
-import { Nav, Platform } from 'ionic-angular';
-import { StatusBar } from '@ionic-native/status-bar';
-import { SplashScreen } from '@ionic-native/splash-screen';
-
-import { HomePage } from '../pages/home/home';
-
-import { SearchPage } from '../pages/search/search';
+import {Component, ViewChild} from "@angular/core";
```
Comment (aagavin) [inline src/app/app.component.ts:1]: can `"@angular/core" ` be `'@angular/core'`
so can we have all single quotes

**Comment by LeontiBrechko** [comment]
Closes issues #5 and #9

**Review by aagavin — CHANGES_REQUESTED** [review]

**Review by aagavin — APPROVED** [review]

File: src/app/app.component.ts (hunk 1 above)
Comment (LeontiBrechko) [reply src/app/app.component.ts:1]: sure, it was done automatically by IDE. Sorry

**Pull request merged**